openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -446,6 +446,11 @@ export default function init(){
         e.preventDefault();
     });
 
+    // For Book Sponsorship Button
+    $(document).ready(function(){
+        $(".cta-btn--sponsor").colorbox({inline: true, opacity: "0.5", href: "#sponsorshipModal"})
```

Comment:
Where is `.cta-btn--sponsor` within the code? Where are the styles? Is that already merged into `master-sponsorship`?